portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,18 +52,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     List<Comment> comments = new ArrayList<>();
+    
     for (Entity entity : results.asIterable()) {
         long id = entity.getKey().getId();
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
 
-        Comment comment = new Comment(id, text, timestamp);
+        translation = translate.translate(text, Translate.TranslateOption.targetLanguage("es"));
```

Comment:
nit: you can make "es" a constant (e.g. `public static final String SPANISH_LANGUAGE_CODE = "es"`)